### PR TITLE
Adds Truncate function

### PR DESCRIPTION
**Pull request by cesartw**
- "Left(s)" - "..."
- Changes SetCSP argument to an array to guarantee output

### CODE REVIEW

File: stringutil/stringutil.go [hunk 1]
```diff
@@ -6,6 +6,19 @@ import (
 	"strings"
 )
 
+// Left returns the "n" left characters of the string.
```
Comment (rafaeljusto) [inline stringutil/stringutil.go:9]:
```suggestion
// Truncate returns the "n" left characters of the string.
```

**Comment by ripexz** [comment]
Would be good to fix the build while you're at it 😉

**Comment by cesartw** [comment]
It only took a little over a year but finally fixed the flaky tests that by looking at the build history in travis this library suffer from 
@ripexz @rafaeljusto 
```
-> Pinging Codecov
    -> Uploading
    X> Failed to upload
    -> Sleeping for 30s and trying again...
    -> Pinging Codecov
    -> Uploading
    X> Failed to upload
    -> Sleeping for 30s and trying again...
    -> Pinging Codecov
    -> Uploading
    X> Failed to upload
    -> Sleeping for 30s and trying again...
    -> Pinging Codecov
    -> Uploading
    X> Failed to upload
    -> Sleeping for 30s and trying again...
    -> Uploading to Codecov
    {"message": "Coverage reports upload successfully", "uploaded": true, "queued": true, "id": "8cdde9a4-6044-45ff-bf25-b1012e885327", "url": "https://codecov.io/github/Teamwork/utils/commit/1a72cad1a75da31ebff9aa2f1edc70d11928d3d7"}
``` 
is still annoying but next year bois

File: stringutil/stringutil.go [hunk 2]
```diff
@@ -7,6 +7,19 @@ import (
 	"unicode"
 )
 
+// Truncate returns the "n" left characters of the string.
+func Truncate(s string, n int) string {
+	if n <= 0 {
+		return ""
+	}
+
+	if len([]rune(s)) <= n {
```
Comment (rafaeljusto) [inline stringutil/stringutil.go:16]: [RuneCountInString](https://pkg.go.dev/unicode/utf8#RuneCountInString) is faster than doing the slice conversion.

```suggestion
	if utf8.RuneCountInString(s) <= n {
```

**Review by rafaeljusto — APPROVED** [review]

**Pull request merged**